src/python/pants/backend/python/providers/python_build_standalone/rules.py

```diff
@@ -204,25 +209,12 @@ async def get_python(
         ),
     )
 
-    await Get(
-        ProcessResult,
-        Process(
-            [
-                cp.path,
-                "--recursive",
-                "--no-clobber",
-                "python",
-                f"{PBS_SANDBOX_NAME}/{python_version}",
-            ],
-            level=LogLevel.DEBUG,
-            input_digest=downloaded_python.digest,
-            description=f"Install Python {python_version}",
-            append_only_caches=PBS_APPEND_ONLY_CACHES,
-            # Don't cache, we want this to always be run so that we can assume for the rest of the
-            # session the named_cache destination for this Python is valid, as the Python ecosystem
-            # mainly assumes absolute paths for Python interpreters.
-            cache_scope=ProcessCacheScope.PER_SESSION,
-        ),
+    # Copy the python binary into the sandbox
+    await async_utils.copytree(
```

Comment:
@cburroughs @benjyw I added a util to perform an async copytree action (adapted from https://github.com/kumaraditya303/aioshutil, since I didn't feel like we needed to pull in an additional package just to get this behavior) - the `async_wrap` util should be useful in other situations like this

Reply:
Sounds good, I don't like pulling in third-party deps for such things.